system/jlib/jlog.cpp

```diff
@@ -2647,7 +2647,7 @@ MODULE_EXIT()
     thePassNoneFilter = nullptr;
     thePassLocalFilter = nullptr;
     thePassAllFilter = nullptr;
-    delete logAccessor.queryExisting();
+    logAccessor.destroy([](IRemoteLogAccess * _logAccessor) { ::Release(_logAccessor); });
```

Comment:
@rpastrana this was previously wrong (should have called Release rather than delete), but the compiler hadn't picked up on it.

Reply:
Yep, understood. This was following the pre-established pattern which your PR corrected.